### PR TITLE
[feature] SearchFilters: adding disableAdvanced property basic mode only

### PR DESCRIPTION
**Pull request by rma4ok**
Adding ability to hide advanced mode option

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 065e56e18a57dd8c0685818802b855272beb0e83
</p>

File: src/SearchFilters.js [hunk 1]
```diff
@@ -77,13 +85,16 @@ let BuilderSearchFilters = ({ setMode, trees, BuilderFilters }) => (
 let SearchFilters = ({
   mode,
   setMode,
+  disableAdvanced,
```
Comment (ccalbaugh) [inline src/SearchFilters.js:88]: I think it's ok if we add the extra word here for clarity and just call it `disableAdvancedSearch`

**Review by ccalbaugh — APPROVED** [review]
Overall looks good!  I just have one nit-picky comment on the name, but I'll approve regardless.

File: src/SearchFilters.js (hunk 1 above)
Comment (rma4ok) [reply src/SearchFilters.js:88]: Yeah good point, `disableAdvancedMode` to be consistent with other props

**Review by chris110408 — APPROVED** [review]
LGTM

File: src/SearchFilters.js (hunk 1 above)
Comment (ccalbaugh) [reply src/SearchFilters.js]: Yeah that sounds good 👍

**Pull request merged**